.github/workflows/docker.yaml

```diff
@@ -0,0 +1,21 @@
+name: DockerImage 
+
+on: [workflow_call]
+
+jobs:
+  build-docker-image-x86_64-job:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
+    - name: Install bazel
+      run: sudo bash tools/buildutils/installbazel.sh
+    - name: bazel version
+      run: bazel version
+    - name: Build docker image
+      run: bazel build --sandbox_writable_path=$HOME //docker:orchestration_image_tar
+    - name: Publish docker image
```

Comment:
It's ok to test building the docker image in presubmit, but publishing the image on presubmit is not necessary. we should publish the image just once after merging.

Reply:
Removed calling logic from presubmit. However I think it's better to be in distinguished yaml files for easier testing.